files/build_templates/qos_config.j2

```diff
@@ -321,21 +328,23 @@
 {% else %}
             "tc_to_queue_map" : "AZURE",
 {% endif %}
-            "tc_to_pg_map"    : "AZURE",
-            "pfc_to_queue_map": "AZURE",
 {% if asic_type in pfc_to_pg_map_supported_asics %}
 {% if port in port_names_list_extra_queues %}
             "pfc_to_pg_map"   : "AZURE_DUALTOR",
 {% else %}
             "pfc_to_pg_map"   : "AZURE",
 {% endif %}
 {% endif %}
+{% if port not in PORT_DPC %}
 {% if port in port_names_list_extra_queues %}
             "pfc_enable"      : "2,3,4,6",
 {% else %}
             "pfc_enable"      : "3,4",
 {% endif %}
-            "pfcwd_sw_enable" : "3,4"
+            "pfcwd_sw_enable" : "3,4",
+{% endif %}
+            "tc_to_pg_map"    : "AZURE",
```

Comment:
tc_to_pg_map and pfc_to_queue_map is also not required for internal ports. Is this excluded?

Reply:
tc_to_pg_map is required for internal ports.

You mean `pfc_to_pg_map` and `pfc_to_queue_map`? Then yes, these two are not required but however having them wouldn't cause any issues IMO. We are configuring the map without enabling pfc on any of the pg's. Let me know if you think i should remove these two a well

Reply:
I agree with @vivekrnv. It's harmless to keep these mappings the same between internal ports and normal ports but the logic will be much simpler.

Reply:
@vivekrnv @stephenxs lossless buffer profiles are not attached to internal ports, so lossless functionality is not possible on these internal ports.

If it is to keep the config uniform across all the ports, this is good.

Reply:
Yes, the idea is to keep the maps uniform across all the ports

Reply:
Hi @kperumalbfn, Can we sign this off?